ports/rp2/mpconfigport.h

```diff
@@ -52,6 +52,11 @@
 #ifndef MICROPY_HW_USB_MSC
 #define MICROPY_HW_USB_MSC (0)
 #endif
+
+#ifndef MICROPY_HW_ENABLE_USB_RUNTIME_DEVICE
+// Enable to support machine.USBDevice
+#define MICROPY_HW_ENABLE_USB_RUNTIME_DEVICE    (0)
```

Comment:
IMO we should enable this by default.  If the feature is not used then users will not notice any difference.